### ISSUE TITLE
Request: add option "run as admin" to handle cases that can't be recorded with keys and handle unicode typing too

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [MathewSachin/Captura#523]
**Is your feature request related to a problem? Please describe.**
Yes. If you run Android emulator, and try to record the screen, it won't show any keys being typed.
Also, if you switch to Hebrew, you won't see Hebrew being typed, but English instead.

**Describe the solution you'd like**
Add option to run as admin. Seems this fixes the issue for the emulator.

**Describe alternatives you've considered**
As written, run as admin.
Sadly, this doesn't solve the issue of Hebrew.

**Additional context**
You can try Android emulator here:
https://developer.android.com/studio/run/emulator

[2020-04-09-12-23-29.zip](https://github.com/MathewSachin/Captura/files/4455220/2020-04-09-12-23-29.zip)

Another issue that I've noticed (shown on the video on 0:07) is that it doesn't show Hebrew characters. I type something in Hebrew, and it shows in English instead

**Comment by MathewSachin** [comment]
You can run as Administrator just by right-clicking on Window. This is not something that should go into the app I think. I'll leave it to the user's to decide whether they want Captura to run as Administrator or not.
If you want, you can create a shortcut to Captura, then go to Properties / Advanced and turn on Run as administrator.

Regarding Hebrew characters not showing, you'll have to create a Keymap file for Hebrew keyboard.
See #510 for more info.

**Comment by AndroidDeveloperLB** [comment]
The thing is that it won't work without admin. Please add it as an option in the settings. It took me time to figure out this is a good workaround. 
Other users might just assume the app can't handle this case.
I think you should offer it, just like I noticed on another app, that they have this setting. 
Only after I tried it here too, I noticed it helps.

As for Hebrew, I don't understand. If you wish, here's how a Hebrew-English keyboard looks like:
https://he.wikipedia.org/wiki/%D7%A7%D7%95%D7%91%D7%A5:Hebrew_keyboard.jpg
